Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "elba-backend"
```

Comment:
This should probably be `elba-website`; eventually this repo will house the front- and backend of the website

Reply:
The rust part of this repo is indeed a backend, and I'm going to build a SPA frontend, thus the rust part would not involves any frontend code.